mdaviz/chartview.py

```diff
@@ -0,0 +1,111 @@
+"""
+Charting widget
+"""
+
+import datetime
+from itertools import cycle
+
+from PyQt5 import QtWidgets
+import pyqtgraph as pg
+
+TIMESTAMP_LIMIT = datetime.datetime.fromisoformat("1990-01-01").timestamp()
+
+# https://pyqtgraph.readthedocs.io/en/latest/api_reference/graphicsItems/scatterplotitem.html#pyqtgraph.ScatterPlotItem.setSymbol
+# https://developer.mozilla.org/en-US/docs/Web/CSS/named-color
+# Do NOT sort these colors alphabetically!  There should be obvious
+# contrast between adjacent colors.
+PLOT_PENS = """
+    r g b c m
+    goldenrod
+    lime
+    orange
+    blueviolet
+    brown
+    teal
+    olive
+    lightcoral
+    cornflowerblue
+    forestgreen
+    salmon
+""".split()
+PLOT_SYMBOLS = """o + x star s d t t2 t3""".split()
+
+pg.setConfigOption("background", "w")
+pg.setConfigOption("foreground", "k")
+GRID_OPACITY = 0.1
+
+
+# Use: next(auto_pen) & next(auto_symbol)
+auto_pen = cycle(PLOT_PENS)
```

Comment:
I update to make it easier to use these lists.

Reply:
See https://github.com/BCDA-APS/gemviz/blob/c750529cf5f3016bf6fdfc49f556c9e4fab64c84/gemviz/chartview.py#L39-L54